gitcha_test.go

```diff
@@ -29,6 +31,12 @@ func TestGitRepoForPath(t *testing.T) {
 }
 
 func TestFindFiles(t *testing.T) {
+	tlink, err := tempLink(".")
+	if err != nil {
+		t.Error(err)
```

Comment:
This should probably be a `t.Fatal`. If `tempLink` returns an error, we wouldn't want this to continue, especially since we call something like `os.Remove` with an undefined value `tlink` next.